boards/arm/tdk_robokit1/tdk_robokit1-common.dtsi

```diff
@@ -81,7 +81,8 @@
 &spi0 {
 	pinctrl-0 = <&spi0_default>;
 	pinctrl-names = "default";
-
+	dmas = <&xdmac 0 DMA_PERID_SPI0_TX>, <&xdmac 1 DMA_PERID_SPI0_RX>;
+	dma-names = "tx", "rx";
```

Comment:
This is OK but maybe it is necessary add an alias for dma as https://github.com/zephyrproject-rtos/zephyr/blob/78809549ee4cf7de3150ff4c3887011825584547/tests/drivers/dma/scatter_gather/boards/frdm_k64f.overlay#L7-L11

and activate DMA function:
```
&dmac {
	status = "okay";
};
```

Reply:
I'm not sure I see the need to alias the DMA here, the board uses a specific dma in this case